app.vue

```diff
@@ -1,23 +1,10 @@
+<script lang="ts" setup></script>
+
 <template>
-  <div>
-    <h1 class="title">Welcome to Constant Reminders</h1>
-    <p id="electron-status">isElectron: {{ useElectron().isElectron }}</p>
```

Comment:
Not sure if we're going to add this back at some point. I know it was throwing an error, before...

Reply:
Naw, this was just for debug purposes